schema/mysql/upgrades/1.0.0.sql

```diff
@@ -26,6 +26,23 @@ ALTER TABLE flapping_history
 ALTER TABLE state_history
   ADD index idx_state_history_event_time (event_time) COMMENT 'Filter for history retention';
 
+ALTER TABLE icon_image
+  DROP COLUMN id,
+  DROP PRIMARY KEY,
+  ADD COLUMN id binary(20) NOT NULL COMMENT 'sha1(environment.id + icon_image)' FIRST,
+  ADD PRIMARY KEY (id);
+
+ALTER TABLE action_url
+  DROP COLUMN id,
+  DROP PRIMARY KEY,
+  ADD COLUMN id binary(20) NOT NULL COMMENT 'sha1(environment.id + action_url)' FIRST,
+  ADD PRIMARY KEY (id);
+
+ALTER TABLE notes_url
+  DROP COLUMN id,
+  DROP PRIMARY KEY,
+  ADD COLUMN id binary(20) NOT NULL COMMENT 'sha1(environment.id + notes_url)' FIRST,
+  ADD PRIMARY KEY (id);
```

Comment:
Will this work if there is data in the tables?

Reply:
Nope, they will lose their hash IDs. Fixed!